tests/apollo/test_skvbc_restart_recovery.py

```diff
@@ -84,25 +88,31 @@ async def test_restarting_replica_with_client_load(self, bft_network):
         # uncomment for live tracking of log messages from the test
         # log = foo()
 
-        for v in range(300):
+        async def client_load(task_status=trio.TASK_STATUS_IGNORED):
+            with trio.CancelScope() as scope:
+                task_status.started(scope)
+                await skvbc.send_indefinite_ops()
+
+        for v in range(loops * 100):
             async with trio.open_nursery() as nursery:
+                scoped_client_load = await nursery.start(client_load)
```

Comment:
Wouldn't it be cleaner to start the nursery outside of the main loop? If I understand correctly, it's a sort of constant background input rate, which doesn't depend on where you are in the iteration.

Reply:
Yep, corrected.

Reply:
Thanks.